### PR TITLE
LOM-300 Accessibility fixes

### PR DESCRIPTION
**Pull request by teroelonen**
# [LOM-300](https://helsinkisolutionoffice.atlassian.net/browse/LOM-300)

## What was done

* Adds more comprehensive is-hidden class for hdbt.
* Enables the refresh-icon to be used on the theme.

## How to install

* Make sure your form-tool instance is up and running on LOM-300_accessibility_fixes branch.
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-LOM-300_accessibility_fixes`
* Run `make drush-cr`

## How to test

* [ ] Check the testing instructions from LOM-300 PR on the form-tool repository.
* [ ] Check that code follows our standards

## Other PRs

* https://github.com/City-of-Helsinki/drupal-helfi-form-tool/pull/84

[LOM-300]: https://helsinkisolutionoffice.atlassian.net/browse/LOM-300?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/misc/icon.twig [hunk 1]
```diff
@@ -1 +1,7 @@
-<span class="hel-icon hel-icon--{{ icon }} {{ class }}" {% if label %} aria-label="{{ label }}" {% else %} aria-hidden="true" {% endif %}></span>
+{% set icon_labelled_by = "hdbt-icon--" ~ random() %}
+
+<span class="hel-icon hel-icon--{{ icon }} {{ class }}" role="img" {% if label %} aria-labelledby="{{ icon_labelled_by }}" {% else %} aria-hidden="true" {% endif %}>
+  {% if label %}
+    <span class="visually-hidden" id="{{ icon_labelled_by }}">{{ label }}</span>
```
Comment (Arkkimaagi) [inline templates/misc/icon.twig:5]: Should this be `is-hidden` instead as `aria-labelledby` should work just as well with `display:none`?

**Review by Arkkimaagi — CHANGES_REQUESTED** [review]
Looks good, just one thing to ponder before we proceed. 🤔

File: templates/misc/icon.twig (hunk 1 above)
Comment (teroelonen) [reply templates/misc/icon.twig]: This is switched now.

**Review by Arkkimaagi — APPROVED** [review]
Thanks! Good job! 🚀

**Pull request merged**